### PR TITLE
date opening issue

### PR DESCRIPTION
**Pull request by srikanthgudipudi**
seva ticket date issue

### CODE REVIEW

File: src/app/about/about.page.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { NetworkService } from '../network.service';
-import { AppConfigs } from '../app.config';
+import { AppConfigs } from '../core-module/constants/api.config';
```
Comment (vishwanath1004) [inline src/app/about/about.page.ts]: File name should be app.config, not api.config.

File: src/app/api/api.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { HomeService } from '../home/home.service';
 import { NetworkService } from '../network.service';
 import { ToastService } from '../toast.service';
-import { AppConfigs } from '../core-module/constants/app.config';
+import { AppConfigs } from '../core-module/constants/api.config'
```
Comment (vishwanath1004) [inline src/app/api/api.ts]: File name should be app.config, not api.config. make changes in all places.

File: src/app/create-task/create-task.page.ts [hunk 3]
```diff
@@ -57,6 +57,8 @@ export class CreateTaskPage implements OnInit {
     this.prepareForm();
     this.today = this.datepipe.transform(this.today, 'dd-MM-yyyy');
   }
+
```
Comment (vishwanath1004) [inline src/app/create-task/create-task.page.ts]: Remove unwanted places.

File: src/app/about/about.page.ts (hunk 1 above)
Comment (srikanthgudipudi) [reply src/app/about/about.page.ts]: changed

File: src/app/api/api.ts (hunk 2 above)
Comment (srikanthgudipudi) [reply src/app/api/api.ts]: changes in all places

File: src/app/create-task/create-task.page.ts (hunk 3 above)
Comment (srikanthgudipudi) [reply src/app/create-task/create-task.page.ts]: removed

**Pull request merged**